src/main/java/com/bina/varsim/tools/evaluation/VCFcompare.java

```diff
@@ -121,11 +121,11 @@ private void canonicalizeToIndels(List<Variant> canonicalVariantList, int[] alle
                     if (end) {
                         canonicalVariantList.add(new Variant(variant.getChr(), currentPosition + reference.length, 0, new byte[0],
                                 new FlexSeq[]{new FlexSeq(Arrays.copyOfRange(alternativeAlleles[0], 0, alleleLengthDifference[0]))},
-                                phase, true, variant.getVariantId(), ".", ""));
+                                phase, true, variant.getVariantId(), VCFparser.DOT, ""));
```

Comment:
DOT may not be a good name. Which field here is being set to dot? For example, EMPTY_QUAL could be "." in which case it should be called EMPTY_QUAL.